e2e/help-button.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test, expect, type Page } from '@playwright/test';
+
+test.describe('help-button tests for a page with three links', () => {
+  let page: Page;
+  // visit the page with the video link
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    await page.goto(
+      '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements'
+    );
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('should render the button, menu and the three links when video is available', async () => {
+    //The button is visible
+    const helpButton = page.getByTestId('get-help-dropdown');
```

Comment:
@Fahdkassim 

I can see we are repeating the same code in both the tests. 
I suggest we move the implementation in a function here and keep it DRY.

Reply:
Done